DESCRIPTION

```diff
@@ -28,10 +28,14 @@ Imports:
 Suggests:
     covr,
     crayon,
+    lintr,
+    tidyverse,
```

Comment:
Since the list of packages changes relatively rarely could we just inline into the package at build-time? Or have a static back up list and then use the actual list if tidyverse is installed? 

Also possible I'm over thinking this.

Reply:
It is a conditional dependency, could potentially only cause issues with the CI, but the builds still seem acceptably fast due package binaries and caching (e.g. https://github.com/r-lib/lifecycle/runs/2176110653).

If users want to run this function themselves they would need all of the packages installed locally anyway.